e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -779,8 +779,15 @@ if (-not (docker images -q aziotbld/testproxy))
 # openssl is currently unavailable as an official release via Chocolatey, so it is advised to perform a manual install from a secured source.
 ############################################################################################################################
 
-Write-Host "Ensure that openssl is available on your system and is set to PATH variable."
-Write-Host "If you have Git installed, openssl can be found at `"<Git_install_directory>\Git\usr\bin\openssl.exe`""
+try
+{
+    Get-Command openssl.exe
+}
+catch
+{
+    Write-Host -ForegroundColor Red "E2E tests require openssl to be installed on your system and set to PATH variable."
```

Comment:
Highlighting this as regular console output since we don't want the script to fail as a result of this but rather only inform the user.